crates/sui-graphql-e2e-tests/tests/stable/consistency/dynamic_fields/immutable_dof.exp

```diff
@@ -144,9 +144,7 @@ Response: {
   "data": {
     "object": {
       "owner": {
-        "parent": {
-          "address": "0x9958869e2da5c5af828a334520a81de2bec0861e171b575db12fb4987cb27a78"
-        }
+        "parent": null
```

Comment:
Likewise -- this seems unexpected.

Reply:
This one is a bit more nuanced -- the parent is a wrapped object, and because of this change, we can now no longer get its ID. This will require further schema changes to accommodate, I think.

Reply:
afaik we overall lack support for wrapped objects like their type, children etc. today, it can be a sizable change across many RPC surfaces if we want to be consistent.